Expense/expenseRouter.js

```diff
@@ -0,0 +1,40 @@
+const express = require("express");
+
+const Expense = require("./Expenses");
+
+const router = express.Router();
+
+router
+  .route("/")
+  .get((req, res) => {
+    Expense.find()
+      .populate("budget")
```

Comment:
We can combine both `populate` into one, and we can add a second argument so `populate` only gives us the fields we're interested in:
```
.populate("budget category", "-_id budgetAmount title")
```